homestar-runtime/src/worker.rs

```diff
@@ -352,7 +383,11 @@ where
                                         })
                                 }
                             }
-                        });
+                        }
+                        .instrument({
+                            info_span!("spawn_tasks").or_current()
```

Comment:
renaming this span to have more helpful information than `spawn tasks`.

Spawning workflow tasks or something like this.

Reply:
Maybe add the workflow cid to this span info.